### ISSUE TITLE
README needs heavy rewriting

### ISSUE THREAD

**Issue by jagerman** [oxen-io/oxen-core#1584]
Someone trying to get oxend to build on an ancient machine (i.e. xenial) recently contacted me and reported being quite mislead by some of the info in our README; looking at it we have *some* up-to-date info, but also a ton of outdated stuff that hasn't really changed from when Monero was forked.

**Comment by darcys22** [comment]
Agree, think we also lost some information between dev becoming the main branch and the readme that was on the master branch
